File: 5th semester/MA (Mobile Apps)/labs/2-7/non-native/footwear_shop/lib/model/Brand.dart
enum Brand {
  DR_MARTENS,
  LOUBOUTIN,
  VALENTINO
}

extension BrandExtension on Brand {
  static const strings = {
    Brand.DR_MARTENS: 'Dr. Martens',
    Brand.LOUBOUTIN: 'Christian Louboutin',
    Brand.VALENTINO: 'Valentino Garavani',
  };

  String get string => strings[this];
}

File: V/MA (Mobile Apps)/labs/2-7/native/app/src/main/java/adrian/ma/footwearshop/model/Brand.kt
package adrian.ma.footwearshop.model

import java.io.Serializable

enum class Brand(val string: String) : Serializable {
    DR_MARTENS("Dr. Martens"),
    VALENTINO("Valentino Garavani"),
    LOUBOUTIN("Christian Louboutin")
}
